### PR TITLE
Tech debt - 885

### PR DESCRIPTION
**Pull request by aduffy-ch**
https://companieshouse.atlassian.net/browse/JU-885

update springboot version & other dependencies
remove healthcheck from lb path

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: IaC Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by ch-code-analysis** [comment]
CI: IaC Security warnings found!

File: terraform/groups/ecs-service/locals.tf [hunk 1]
```diff
@@ -10,7 +10,7 @@ locals {
   kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
   lb_listener_rule_priority  = 63
   lb_listener_paths          = ["/transactions/*/confirmation-statement/*", "/confirmation-statement/*", "/private/confirmation-statement/*", "/private/transactions/*/confirmation-statement/*/filings", "/confirmation-statement-api/healthcheck"]
```
Comment (foman-ch) [inline terraform/groups/ecs-service/locals.tf:12]: think it's this line that you need to remove the healtcheck from, in order for it not to be picked up by the load balancer

File: terraform/groups/ecs-service/locals.tf [hunk 2]
```diff
@@ -10,7 +10,7 @@ locals {
...
-  healthcheck_path           = "/confirmation-statement-api/healthcheck"
+  healthcheck_path           = "/confirmation-statement-api"
```
Comment (foman-ch) [inline terraform/groups/ecs-service/locals.tf:13]: keep this line the same - the container will still need the healthcheck to be defined as:/confirmation-statement-api/healthcheck

**Review by foman-ch — CHANGES_REQUESTED** [review]

**Comment by ch-code-analysis** [comment]
CI: IaC Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: IaC Security warnings found!

**Pull request merged**